src/job.rs

```diff
@@ -339,7 +342,21 @@ impl Job {
                 job_try!(success_cb().await);
                 Status::Finished(Ok(()))
             }
+        };
+
+        if let Status::Finished(Err(err)) = &status {
+            // We couldn't send the message, so mark it as failed
+            let msg_id = MsgId::new(self.foreign_id);
+            message::set_msg_failed(context, msg_id, Some(err.to_string())).await;
+            match Message::load_from_db(context, msg_id).await {
+                Ok(message) => chat::add_info_msg(context, message.chat_id, err.to_string()).await,
+                Err(e) => error!(
+                    context,
+                    "couldn't load chat_id to inform user about SMTP error: {}", e
+                ),
+            };
```

Comment:
Has nothing to do with this PR but I think we don't need this info_msg anymore now that at least Android shows the error msg on tap (I don't know about the two Desktop applications and iOS though)

Reply:
On android you just show the message details or how is it working? If i'm on it i can implement the same on desktop too.

Reply:
No, if you tap a failed message, the error is shown (not the rest of the message details, not to confuse users) (you can get it using dc_msg_get_error())